src/web/components/Container/Container.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Container({ children }) {
```

Comment:
That is a great component 😄

Reply:
Thank you. It was an extremely difficult component to make. :smile: